aws/s3-scan-object/iam.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_iam_role" "scan_files" {
+  name               = "ScanFilesGetObjects"
+  assume_role_policy = data.aws_iam_policy_document.scan_files_assume_role.json
+}
+
+data "aws_iam_policy_document" "scan_files_assume_role" {
+  statement {
+    effect = "Allow"
+
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = [var.scan_files_role_arn]
+    }
+  }
+}
+
+resource "aws_iam_policy" "scan_files" {
+  name   = "ScanFilesGetObjects"
+  path   = "/"
+  policy = data.aws_iam_policy_document.scan_files.json
+}
+
+resource "aws_iam_role_policy_attachment" "scan_files" {
+  role       = aws_iam_role.scan_files.name
+  policy_arn = aws_iam_policy.scan_files.arn
+}
+
+data "aws_iam_policy_document" "scan_files" {
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:ListBucket",
+      "s3:GetObject",
+      "s3:GetObjectVersion"
```

Comment:
can you also add getobject tags. this way i can compare md5 of original file and md5 of downloaded file.